hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/schema/TablesStorageTest.java

```diff
@@ -123,6 +124,50 @@ public void when_clusterVersionIs5dot2_then_oldCatalogIsMigratedOnFirstRead() {
         assertThat(storage.oldStorage().size() == 0);
     }
 
+    @Test
+    public void when_clusterVersionIs5dot2_then_oldCatalogIsNotMigratedOnFirstReadAfterInitialization() {
+        String name = randomName();
+        storage.initializeWithListener(new TablesStorage.EntryListenerAdapter() {
+            @Override
+            public void entryUpdated(EntryEvent<String, Object> event) {
+            }
+
+            @Override
+            public void entryRemoved(EntryEvent<String, Object> event) {
+            }
+        });
+        storage.put(name, mapping(name, "type"));
+        storage.oldStorage().putAll(storage.newStorage());
+        storage.newStorage().clear();
+        storage.allObjects();
+
+        assertThat(storage.newStorage().size() == 0);
+        assertThat(storage.oldStorage().size() == 0);
```

Comment:
`oldStorage.size()` is 1 at this point. `assertThat` is used incorrectly.

Overall, I don't like this test. It puts a mapping to new storage, then moves new storage to old storage and clears new storage. Why don't we directly put to old storage? 

I think we can test this:
- put mapping to old storage manually
- assert that allObjects doesn't see it
- assert thta oldStorage still has the object after `allObjects` call

Reply:
Just noticed that `assertThat` is used incorrectly many times in the test :smile:

Reply:
Power of copy paste :) I will fix them all.